frontend/src/components/PsyListing/PsyListing.jsx

```diff
@@ -78,6 +78,10 @@ const PsyListing = () => {
         return false;
       }
 
+      if (languageFilter && !utils.matchFilter(psychologist.languages, languageFilter)) {
```

Comment:
Why ?

Reply:
I moved it before the check on `addressFilter` because if it's `AROUND_ME`, we returned `true` without filtering on language.